mautic/mautic
# Handle incorrect filter

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | Yes
| New feature/enhancement? (use the a.x branch)      | No
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | No
| Automated tests included?              | Yes 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



As we have created custom object the need appeared to handle until now non-existent error.
We need a record of this.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Enable custom object 
3. Create a custom object
4. Create a segment based upon custom object
5. Rebuilt segment
6. Disable the custom object
7. Re-run the segment
8. Check the logs for below line should be there:
`Segment Query is missing table, perhaps incorrectly registered custom Query Builder.`
